### PR TITLE
Merge neon examples

### PR DESCRIPTION
**Pull request by nervetumer**

### CODE REVIEW

File: ImageCaptioning/LSTM/readme.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ found in the following [CVPR-2015 paper](http://arxiv.org/abs/1411.4555):
 
 The model was trained for 15 epochs where 1 epoch is 1 pass over all 5 captions of each image. 
 Training data was shuffled each epoch. To evaluate on the test set, download the model and weights, 
-and run:
+activate the neon virtualenv and ifrom the root neon directory run:
```
Comment (jennifermyers) [inline ImageCaptioning/LSTM/readme.md]: typo ifrom->from

File: ImageClassification/CIFAR10/All_CNN/readme.md [hunk 2]
```diff
@@ -4,42 +4,39 @@ This is an implementation of a deep convolutional neural network model inspired
...
-global contrast normalized data, without crops or flips.  This is the same performance we achieve running the 
-same model configuration and data through Caffe.  
 
-### Instructions
-This script was tested with the 
-[neon commit SHA e7ab2c2e2](https://github.com/NervanaSystems/neon/commit/e7ab2c2e27f113a4d36d17ba8c79546faed7d916).  
-Make sure that your local repo is synced to this commit and run the 
-[installation procedure](http://neon.nervanasys.com/docs/latest/installation.html) before proceeding.
+### neon version
+The model weigth file above has been generated using neon version tag [v1.4.0]((https://github.com/NervanaSystems/neon/releases/tag/v1.4.0).
```
Comment (jennifermyers) [inline ImageClassification/CIFAR10/All_CNN/readme.md]: weigth->weight

File: ImageClassification/CIFAR10/All_CNN/readme.md [hunk 3]
```diff
@@ -4,42 +4,39 @@ This is an implementation of a deep convolutional neural network model inspired
...
+It may not work with other versions.
 
+### Performance
+This model is acheiving 89.5% top-1 accuracy on the validation data set.  This accuracy is
```
Comment (jennifermyers) [inline ImageClassification/CIFAR10/All_CNN/readme.md]: acheiving->achieving (also on following line)

File: ImageClassification/CIFAR10/All_CNN/readme.md [hunk 4]
```diff
@@ -4,42 +4,39 @@ This is an implementation of a deep convolutional neural network model inspired
...
-[neon commit SHA e7ab2c2e2](https://github.com/NervanaSystems/neon/commit/e7ab2c2e27f113a4d36d17ba8c79546faed7d916).  
-Make sure that your local repo is synced to this commit and run the 
-[installation procedure](http://neon.nervanasys.com/docs/latest/installation.html) before proceeding.
+### neon version
+The model weigth file above has been generated using neon version tag [v1.4.0]((https://github.com/NervanaSystems/neon/releases/tag/v1.4.0).
+It may not work with other versions.
 
+### Performance
+This model is acheiving 89.5% top-1 accuracy on the validation data set.  This accuracy is 
+acheived using zca whitened, global contrast normalized data, without crops or flips.
```
Comment (jennifermyers) [inline ImageClassification/CIFAR10/All_CNN/readme.md]: acheived->achieved

File: NLP/QandA/bAbI/readme.md [hunk 5]
```diff
@@ -2,61 +2,25 @@
...
+See the [readme](https://github.com/NervanaSystems/neon/blob/master/examples/babi/README.md)
+there for instructions on how to train the model and run the demo.
 
 ### Trained weights
 The trained weights file for a GRU network trained on task 15 can be downloaded from AWS
 using the following link: [trained model weights on task 15][S3_WEIGHTS_FILE].
 [S3_WEIGHTS_FILE]: https://s3-us-west-1.amazonaws.com/nervana-modelzoo/bAbI/babi_task15.p
 
+### neon version
+The model weigth file above has been generated using neon version tag [v1.4.0]((https://github.com/NervanaSystems/neon/releases/tag/v1.4.0).
```
Comment (jennifermyers) [inline NLP/QandA/bAbI/readme.md]: weigth->weight

File: NLP/SentimentClassification/IMDB/readme.md [hunk 6]
```diff
@@ -4,35 +4,33 @@ This is an implementation of a LSTM model to solve the IMDB sentiment classifica
...
+The model script is included with the neon repo examples ([link](https://github.com/NervanaSystems/neon/blob/master/examples/imdb_lstm.py))
 
 ### Trained weights
 
 The trained weights file can be downloaded from AWS using the following link:
 [imdb_lstm.p][S3_WEIGHTS_FILE].
 [S3_WEIGHTS_FILE]: https://s3-us-west-1.amazonaws.com/nervana-modelzoo/imdb_lstm/imdb_lstm.p
 
+### neon version
+The model weigth file above has been generated using neon version tag [v1.4.0]((https://github.com/NervanaSystems/neon/releases/tag/v1.4.0).
```
Comment (jennifermyers) [inline NLP/SentimentClassification/IMDB/readme.md]: weigth->weight

**Comment by nervetumer** [comment]
This should be ready.  There one model broken because of an issue in neon and on model that needs to be refit (CIFAR10-MSRA)

**Comment by jennifermyers** [comment]
LGTM

**Pull request merged**